python/lsst/dax/imgserv/jobqueue/imageworker.py

```diff
@@ -78,17 +80,30 @@ def get_image_async(self, *args, **kwargs):
     config["DAX_IMG_CONFIG"] = config_path
     meta_url = imgserv_config.webserv_config["dax.imgserv.meta.url"]
     config["DAX_IMG_META_URL"] = meta_url
-    soda = imageSODA.ImageSODA(config)
-    image = soda.do_sync(params)
-    # save the result image in local temp dir
-    with tempfile.NamedTemporaryFile(dir=config["DAX_IMG_TEMPDIR"],
+    image = None; status = "OK"
+    try:
+        soda = imageSODA.ImageSODA(config)
+        image = soda.do_sync(params)
+    except ImageNotFound:
+        status = "Error=Image Not Found"
```

Comment:
Do you want to be hard coding the error message here when the error message will be in the exception itself?